src/components/FeatureSetsPanel/featureSetPanel.util.js

```diff
@@ -108,17 +104,18 @@ export const checkValidation = (
     isValid = false
   }
 
-  if (
-    externalOfflineTarget &&
-    (externalOfflineTarget.path.length === 0 ||
-      !validation.isExternalOfflineTargetPathValid)
-  ) {
-    setValidation(prevState => ({
-      ...prevState,
-      isExternalOfflineTargetPathValid: false
-    }))
+  if (!newFeatureSet.spec.passthrough) {
+    if (
+      externalOfflineTarget &&
+      (externalOfflineTarget.path.length === 0 || !validation.isExternalOfflineTargetPathValid)
+    ) {
+      setValidation(prevState => ({
+        ...prevState,
+        isExternalOfflineTargetPathValid: false
+      }))
```

Comment:
There is no need to have inner `if` statement. You can add those conditions to the single `if` statement